File: src/Services/Applications/Application.Service.Queries/DTOs/AplicacionDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.Service.Queries.DTOs
{
    public class AplicacionDto
    {
        public int id { get; set; }
        public string descripcion{ get; set; }
        public string url { get; set;}
        public string origen { get; set; }
        public string clasificacion_ens { get; set; }

    }
}

File: Verifica/angularapp/src/app/Models/aplicacion.dto.ts
export class AplicacionDto {
  id?: number;
  descripcion: string;
  url: string;
  origen: string; 
  clasificacion_ens: string;

  constructor(
    descripcion: string,
    url: string,
    origen: string,
    clasificacion_ens: string) {
    this.descripcion = descripcion;
    this.url = url;
    this.origen = origen;
    this.clasificacion_ens = clasificacion_ens;
  }
}
